AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -395,6 +395,7 @@ jokerplayer.net##snackbar
 kashi-mo.com##.l-footer > a.wimax_bnr
 afonya-spb.ru##.jointcontent_block
 afonya-spb.ru##.n-js-promoWrapper
+journalstar.com#?#section.block:has(> div.clearfix > div h3:contains(/Print|ads/))
```

Comment:
to css_extended.

Reply:
it is OK. I think we can stop devide ext-css and elemhide rules(and CSS, JS and scriptlets). Without it will be easier to make groups of rules for specific sites. Just don't forget to add `?` to ext-css rules.